parser.cpp

```diff
@@ -159,9 +159,13 @@ void Stripper::SetEntityList(const char *ents)
     char *_tmp = NULL;
     size_t _tmpsize = 0;
     size_t pos = 0;
+    char in_string = 0;
```

Comment:
Yeah, I think I just looked at the other lines and felt like I am in a C environment if that makes sense.
I could change it to bool and ^= true but I don't think it matters tbh. looking at the rest of the code, there is no clear convention and it looks like a C program with C++ thrown in.

My only hope would be that there is some case where this will break on some map though it's been running fine on the GFL Zombie Escape server for the past few days.

I am still wondering how to make a functional build with ambuild, I used the makefile to build the stripper.core.so, when I used ambuild the stripper.16.css.so wouldn't load (see #2 Could not find LoadStripper function)